documentation/source/user_section/installation/linux.rst

```diff
@@ -46,13 +46,14 @@ Installation Options
 Option 1: Install from Package (recommended)
 --------------------------------------------
 
-The simplest way to install SCT is to do it via a stable release. First, download the `latest release <https://github.com/spinalcordtoolbox/spinalcordtoolbox/releases>`_. Major changes to each release are listed in the :doc:`/dev_section/CHANGES`.
+The simplest way to install SCT is to do it via a stable release. First, navigate to the `latest release <https://github.com/spinalcordtoolbox/spinalcordtoolbox/releases>`_, then download the install script for SCT (``install_sct-<version>_linux.sh``). Major changes to each release are listed in the :doc:`/dev_section/CHANGES`.
 
-Once you have downloaded SCT, unpack it (note: Safari will automatically unzip it). Then, open a new Terminal, go into the created folder and launch the installer:
+Once you have downloaded SCT, open a new Terminal in the location of the downloaded script, then launch the installer. For example, if the script was downloaded to `Downloads/`, then you would run:
 
 .. code:: sh
 
-  ./install_sct
+  cd ~/Downloads
+  ./install_sct-<version.sh
```

Comment:
> Does this need a `chmod +x install_sct<version>_linux.sh`? I'm not sure if the downloaded file would have executable permissions by default.

Good question! I'm going to fork SCT and try creating a new release to see what happens.

> Also, the closing angle bracket and OS name are missing.

Done!

Reply:
You were absolutely correct that `chmod +x` (or `bash`) is needed. _((6.0 release on fork [here](https://github.com/joshuacwnewton/spinalcordtoolbox/releases/tag/untagged-a3dd968a330fc7e6458c)))_

Reply:
I've chosen `bash`, since I like that keeps things a one-liner.

But, a random thought: Will this in any way interact with systems that have a non-bash shell as their default?

I presume not, because `install_sct` has `bash` in its shebang, so `bash` was always being used anyway? And the value of `$SHELL` (which we use to determine the default shell) doesn't change if a script is invoked via `bash`?

Reply:
Good question! I hadn't noticed that we use the `$SHELL` variable in the script. You're right that calling `bash install_sct` should have the same effect as `./install_sct`, given that the shebang line specifies `bash`. Also, according to the `bash` manual page:
>        SHELL  This variable expands to the full pathname to the shell.  If  it
>              is  not  set  when the shell starts, bash assigns to it the full
>             pathname of the current user's login shell.

So we're doubly covered: normally the user's shell will already have `$SHELL` set in the environment, and even if not, `bash` will look up the user's login shell.